src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -528,4 +470,124 @@ protected boolean matchesSafely(URI uri) {
       }
     };
   }
+
+  @Test
+  public void testSuccessfulIngestRequest() throws Exception {
+    AtomicReference<String> metacardId = new AtomicReference<>();
```

Comment:
It looks like this var refers to a URI/URL. Calling it `metacardUri` or similar would help readers know what to expect.

Reply:
Updated to `datasetUri` 👍

Reply:
We should use `datasetId`. https://github.com/connexta/ion-transformation-api/pull/68#discussion_r345861497